### PR TITLE
703 additional fields topic config

### PR DESCRIPTION
**Pull request by dcj2489**
- [ ] **Breaking change?**  -> No

**What changes did you make?** 
Added some new fields to Topic Configs.

**Is there anything you'd like reviewers to focus on?**
New fields on TopicConfig

**How Has This Been Tested?**
- [ ] No need to
- [x] Manually(please, describe, when necessary)
- [x] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** 
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings(e.g. Sonar is happy)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

### CODE REVIEW

File: kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaConsumerTests.java [hunk 1]
```diff
@@ -126,4 +127,29 @@ public void shouldReturn404ForNonExistingTopic() {
...
+            .exchange()
+            .expectStatus()
+            .isOk();
+
+    WebTestClient.ListBodySpec result = webTestClient.get()
+            .uri("/api/clusters/{clusterName}/topics/{topicName}/config", LOCAL, topicName)
+            .exchange()
+            .expectStatus()
+            .isOk()
+            .expectBodyList(TopicConfig.class);
```
Comment (IldarAlmakaev) [inline kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaConsumerTests.java:152]: Could you please validate the actual response with an expected one including `sourceConfig`, `isSensitive` and `isReadOnly` properties

**Review by IldarAlmakaev — CHANGES_REQUESTED** [review]

**Review by MarselAhmetov — CHANGES_REQUESTED** [review]
Please check last changes in master, I made similar changes.
Here you will always get empty synonyms list you have to map it correctly, also please use classes created in #730 PR

File: kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml [hunk 2]
```diff
@@ -1551,9 +1551,40 @@ components:
...
+    Source:
+      type: string
+      enum:
+        - DYNAMIC_TOPIC_CONFIG
+        - DYNAMIC_BROKER_LOGGER_CONFIG
+        - DYNAMIC_BROKER_CONFIG
+        - DYNAMIC_DEFAULT_BROKER_CONFIG
+        - STATIC_BROKER_CONFIG
+        - DEFAULT_CONFIG
+        - UNKNOWN
```
Comment (MarselAhmetov) [inline kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml:1586]: Please merge master to your branch, I already added such classes there :)

File: kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaConsumerTests.java (hunk 1 above)
Comment (dcj2489) [reply kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaConsumerTests.java]: Hey @IldarAlmakaev , I updated the tests

File: kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml (hunk 2 above)
Comment (dcj2489) [reply kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml]: Done, I had to resolve some conflicts, but I'm now using your classes

**Comment by MarselAhmetov** [comment]
According to documentation for getting config with synonyms you have to add an option in `loadTopicsConfig`
```
adminClient.describeConfigs(resources, new DescribeConfigsOptions().includeSynonyms(true)).all()
```

**Comment by dcj2489** [comment]
> According to documentation for getting config with synonyms you have to add an option in `loadTopicsConfig`
> 
> ```
> adminClient.describeConfigs(resources, new DescribeConfigsOptions().includeSynonyms(true)).all()
> ```

Done

**Review by MarselAhmetov — APPROVED** [review]

**Review by tdavletov — APPROVED** [review]

File: kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaConsumerTests.java [hunk 3]
```diff
@@ -164,4 +169,37 @@ public void shouldReturn404ForNonExistingBroker() {
...
+            .expectStatus()
+            .isOk()
+            .expectBodyList(TopicConfig.class)
+            .returnResult()
+            .getResponseBody();
+
+    assert configs != null;
+    assert !configs.isEmpty();
+    Assertions.assertNotEquals(null, configs.get(0).getName());
+    Assertions.assertNotEquals(null, configs.get(0).getIsReadOnly());
```
Comment (IldarAlmakaev) [inline kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaConsumerTests.java:201]: I believe there should be a method like `assertNotNull(...)` Could you please use it?

**Review by IldarAlmakaev — CHANGES_REQUESTED** [review]

File: kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaConsumerTests.java (hunk 3 above)
Comment (dcj2489) [reply kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaConsumerTests.java]: This is done

**Pull request merged**